package.json

```diff
@@ -1,9 +1,6 @@
 {
   "name": "aws-lambda-powertools-python-e2e",
   "version": "1.0.0",
-  "devDependencies": {
-    "aws-cdk": "^2.99.1"
-  },
```

Comment:
Is it right?

Reply:
Interesting, I don't even know what caused this change. I'll revert